ghostfolio/docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
       APP_PORT: 3334
 
   server:
-    image: ghostfolio/ghostfolio:1.241.0@sha256:0fd0b0bd722420bcc8fb92076f4bcfd8e1dcfb9b8b136367539e82184e992e1c
+    image: ghostfolio/ghostfolio:1.269.0@sha256:096ea10a7e9c8c471701da78d3aeb8434879e465b36b3564d6bb424bc47c2b3d
```

Comment:
I updated this with the sha256 digest for the multi-arch Docker image. It needs to be the multi-arch digest so that the app can run on Umbrel's installed on both arm64 and x86 devices. You can get the multi-arch digest by running:
`docker pull ghostfolio/ghostfolio:1.269.0`, replacing `1.269.0` with whichever tag you are grabbing and the output of that command will include the multi-arch digest.